docs/source/DeveloperGuide/AddingNewComponents/Utilities/gLogger/gLogger/Basics/index.rst

```diff
@@ -0,0 +1,607 @@
+*Logging* name and the level of the message, like this:
+
+::
+
+    [Date] UTC [System]/[Component]/[Log][Path]:[Line] [Level]: [Message]
+    2017-04-25 15:51:01 UTC Framework/Atom/log[opt/dirac/DIRAC/FrameworkSystem/private/standardLogging/Logging.py:325]INFO: message
+
+It is composed by the caller object path and the line of the call. This
+option requires that you set *LogShowLine* at *True* in the *cfg* file
+and the root *Logging* level to *DEBUG* in order to be effective:
+
+::
+
+    ShowLogLine = True
+    LogLevel = DEBUG
+
+As the *threadIDIsShown* option, the *headerIsShown* boolean has to be
+at *True* too. Moreover, it is totally possible to have the thread ID
+and the caller path name at the same time.
+
+Nevertheless, this option is useless because always displays the same
+caller. It can be explained due to the *Logging* object which wrap the
+original Python *logging* library.
+
+Remove colors on the log records
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+*LogColor* option is only available from the *cfg* file too, and only
+for the *stdout* and the *stderr* with agents, services and executors.
+By default, the *LogColor* option is set a *True* and adds colors on the
+log records according to their levels. You can remove colors setting the
+option at *False* in the *cfg* file:
+
+::
+
+    LogColor = False
+
+We can find a *cfg* file example containing different options in the `cfg file example`_ part.
+
+Get the option values
+~~~~~~~~~~~~~~~~~~~~~
+
+It is possible to obtain the names and the values associated of all
+these options with the *getDisplayOptions* method. This method returns
+the dictionary used by the *Logging* object itself and not a copy, so we
+have to be careful with its use. Here is an example:
+
+::
+
+    logger = gLogger.getSubLogger("logger")
+    logger.getDisplayOptions()
+    # > {'Color': False, 'Path': False, 
+    #    'headerIsShown': True, 'threadIsShown': False}
+
+Send a log record in different outputs
+--------------------------------------
+
+Backend presentation
+~~~~~~~~~~~~~~~~~~~~
+
+*Backend* objects are used to receive the log record created before,
+format it according to the choice of the client, and send it in the
+right output. Currently, there are four different *Backend* object
+inherited from a base. Here is a table presenting them:
+
++-----------------+-------------------------+
+| Backend name    | Output                  |
++=================+=========================+
+| stdout          | standard output         |
++-----------------+-------------------------+
+| Stderr          | error output            |
++-----------------+-------------------------+
+| RemoteBackend   | SystemLogging service   |
++-----------------+-------------------------+
+| FileBackend     | file                    |
++-----------------+-------------------------+
+
+As we may notice, *gLogger* has already a *stdout Backend* by default.
+
+Add a *Backend* to your *Logging*
```

Comment:
Add a note that the next step is to have it plugable, so that anyone can define a new type of backend.